plugins/vppv2/puntplugin/vppcalls/punt_vppcalls.go

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	"net"
+
+	"github.com/go-errors/errors"
+	ipApi "github.com/ligato/vpp-agent/plugins/vpp/binapi/ip"
```

Comment:
For better clarity use aliases which fully contain the original name and preferably with snake case to have the exact name as the suffix. 

For this case:
- `binapi_ip`
- `binapi_punt`